pkg/status/status.go

```diff
@@ -347,8 +347,12 @@ func (r *reporter) monitorClusterStatus() {
 				}
 				if isSucceeding {
 					statusConditions = conditionListBuilder(configv1.OperatorDegraded, configv1.ConditionFalse, fmt.Sprintf("Current CR sync ratio (%g) meets the expected success ratio (%g)", *ratio, successRatio), "OperandTransitionsSucceeding")
+					statusConditions = conditionListBuilder(configv1.OperatorAvailable, configv1.ConditionTrue, fmt.Sprintf("Available release version: %s", r.version), "OperandTransitionsSucceeding")
```

Comment:
Do we need to set statusConditions each time?

Reply:
A good point. That can be cleaned up a bit.